edgecast/customer/customer_user_models.go

```diff
@@ -0,0 +1,145 @@
+package customer
+
+// CustomerUser represents data for a Customer User that is a child object of
+// the associated Customer. Customer Users are used to login to my.edgecast.com
+// (the MCC) and are associated with API tokens for legacy APIs and oath2
+// clients. For more information on user management, please review the
+// Web Services REST API (Partner) document available at the below url:
+// https://partner.edgecast.com/support/default.aspx
+type CustomerUser struct {
+	// A string identifying the street address for the user.
+	Address1 string
+
+	// A string providing additional location information for the user.
+	// Typically, this parameter is used to define a suite number or an
+	// apartment number.
+	Address2 string
+
+	// A string identifying the city associated with the user's address.
+	City string
+
+	// A string identifying the state associated with the user's address.
+	// If the country associated with this account is "United States," then
+	// valid values for this parameter are restricted to the two character
+	// state abbreviations defined in the State/Province section in Appendix D.
+	// of the Web Services REST API (Partner) document
+	State string
+
+	// A string identifying the ZIP code associated with a user's address.
+	ZIP string `json:"Zip"`
```

Comment:
We should add omitempty by default.

Reply:
@changgyu-oh added this as well to all properties.